app/players/player.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+    nickname = ""
+
+    @abstractmethod
+    def get_rating(self) -> None:
```

Comment:
It returns int
You must specify type you expect in all your child classes will return